packages/shared/providers.yaml

```diff
@@ -3495,6 +3495,23 @@ notion:
             response_path: results
     docs: https://docs.nango.dev/integrations/all/notion
 
+notion-scim:
+    display_name: Notion (SCIM API)
+    categories:
+        - knowledge-base
+        - productivity
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.notion.com/scim
+        headers:
+            Authorization: Bearer ${apiKey}
+    docs: https://docs.nango.dev/integrations/all/notion
+    credentials:
+        apiKey:
+            type: string
+            title: API Key
+            description: The API key for your Notion account
```

Comment:
It's missing validation and verification, otherwise somebody will need to painfully backfill those 😬

Reply:
Yeah. Missing both because we can’t access a Notion enterprise account so I dont know what either look like